tests/ip/test_mgmt_ipv6_only.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+
+from tests.common.fixtures.duthost_utils import convert_and_restore_config_db_to_ipv6_only  # noqa F401
+from tests.common.helpers.assertions import pytest_assert, pytest_require
+
+pytestmark = [
+    pytest.mark.topology('any'),
+    pytest.mark.device_type('vs')
+]
+
+
+def test_bgp_facts_ipv6_only(duthosts, enum_frontend_dut_hostname, enum_asic_index,
```

Comment:
>test_bgp_facts_ipv6_only [](http://example.com/codeflow?start=4&length=24)

Instead of duplicating code, can we use the fixture to auto create a new testcase for ipv6 only?

Reply:
The bgp test and feature test are just for sample sake only.
I tried directly importing test_bgp_facts from the bgp/test_bgp_facts.py and adding the IPv6 conversion fixture, but the bgp test ran twice, one original, and one with IPv6 conversion.
So I think the temporary duplicated code would be the most efficient way to do the verification, I believe we will have a very general IPv6 test case.
Let's remove these two sample test cases at that time.